package.json

```diff
@@ -157,18 +157,28 @@
   "scripts": {
     "vscode:prepublish": "tsc -p ./",
     "compile": "tsc -watch -p ./",
-    "test": "sh scripts/e2e.sh"
+    "test": "sh scripts/e2e.sh",
+    "clean": "rimraf out",
+    "format": "yarn prettier --write .",
+    "lint": "eslint -c .eslintrc.js --ext .ts src test",
+    "build": "npm run clean && npm run lint && npm run vscode:prepublish"
```

Comment:
This list is easier to read, search, and diff if it alphabetical. 
Not a big deal, just my opinion.

Reply:
Also just noticed that this line is using `npm` but above it is you use `yarn`. While you can use both, I think consistency is better.